ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -31,20 +35,30 @@ export default function CollectiblesItems({
   collections = {},
   previouslyOwnedCollection = {},
 }) {
+  const dispatch = useDispatch();
   const collectionsKeys = Object.keys(collections);
+  const collectiblesDropdownState = useSelector(getCollectiblesDropdownState);
+  const previousCollectionKeys = usePrevious(collectionsKeys);
 
-  // if there is only one collection present set it to open when component mounts
-  const [dropdownState, setDropdownState] = useState(() => {
-    return collectionsKeys.length === 1
-      ? {
-          [PREVIOUSLY_OWNED_KEY]: false,
-          [collectionsKeys[0]]: true,
-        }
-      : { [PREVIOUSLY_OWNED_KEY]: false };
-  });
+  useEffect(() => {
+    if (
+      !Object.keys(collectiblesDropdownState).length &&
```

Comment:
`!Object.keys(collectiblesDropdownState).length` feels kinda weird; maybe a `=== 0` check instead?

Reply:
Actually I'm going to merge for now and update in a follow up clean up PR!